src/middlewared/middlewared/plugins/filesystem.py

```diff
@@ -420,6 +513,36 @@ def setperm(self, job, data):
         action = 'clone' if mode else 'strip'
         self._winacl(data['path'], action, uid, gid, options)
 
+    @accepts()
+    async def default_acl_choices(self):
+        """
+        Get list of default ACL types.
+        """
+        return [x.name for x in ACLDefault]
+
+    @accepts(
+        Str('acl_type', default='GENERIC_OPEN', enum=[x.name for x in ACLDefault])
```

Comment:
Some instances of GENERIC still in there.
```suggestion
        Str('acl_type', default='OPEN', enum=[x.name for x in ACLDefault])
```